packages/protobuf-test/typescript/ts4_0_2/tsconfig.json

```diff
@@ -0,0 +1,72 @@
+{
+  "include": ["../../src/**/*.ts"],
+  "exclude": ["../../src/**/*.test.ts", "../../src/gen/**/*.ts"],
+  "compilerOptions": {
+    "explainFiles": true,
+    /* Visit https://aka.ms/tsconfig.json to read more about this file */
+
+    /* Basic Options */
+    // "incremental": true,                   /* Enable incremental compilation */
+    "target": "es5" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019', 'ES2020', or 'ESNEXT'. */,
+    "module": "commonjs" /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', 'es2020', or 'ESNext'. */,
+    // "lib": [],                             /* Specify library files to be included in the compilation. */
+    // "allowJs": true,                       /* Allow javascript files to be compiled. */
+    // "checkJs": true,                       /* Report errors in .js files. */
+    // "jsx": "preserve",                     /* Specify JSX code generation: 'preserve', 'react-native', or 'react'. */
```

Comment:
Those config files should make it clear to the user what we test, and with what options. 

I suggest that we remove all options that are commented out to reduce the file size to a fraction, and add a comment that states explicitly that those are the default options for TypeScript v4.0.2 created with `tsc --init`.